modules/library/api/src/main/java/org/geotools/styling/PolygonSymbolizer.java

```diff
@@ -2,7 +2,7 @@
  *    GeoTools - The Open Source Java GIS Toolkit
  *    http://geotools.org
  * 
- *    (C) 2002-2008, Open Source Geospatial Foundation (OSGeo)
+ *    (C) 2002-2016, Open Source Geospatial Foundation (OSGeo)
```

Comment:
No need to update the copyright dates anymore, just add a standard header with the current year if the file is new or does not have a header